File: Stars Pattern/Right Sided Increasing Triangle/Pattern.py
#!/usr/bin/env python3

"""
Right Sided Increasing Triangle Pattern
"""

from sys import path

path.append("../.")


from ConsoleNumInput import inInt


#### Begin ####

n = inInt().getInput(5)

# Method 1
print("Method 1")

for x in range(n): 
    
    for y in range(x, n):
        print("   ", end="")
    
    for z in range(x + 1):
        print(" * ", end="")
    
    print("")


print("\n*~*~*~*~*~*~*~*~*~*~*~*\n")


# Method 2
print("Method 2")

for x in range(n):
    
    print("   " * (n - x), end="")
    
    print(" * " * (1 + x))




print("")

"""
Author: Jovan De Guia
Github Username: jxmked
"""

File: Stars Pattern/Right Sided Increasing Triangle/Pattern.c
#include <stdio.h>
#include <stdlib.h>

// External File
#include "../ConsoleNumInput.c"

/**
 * Right Sided Increasing Triangle Pattern
 * */

static int loop(int n);


int main(void){
    // Default
    int n = 5;
    
    do {
        // From External File
        clrscr();
        
        loop(n);
        
        n = inInt();
        
        // This will never trigger. 
        // System required. Hahaha
        if(! n){
            return 0;
        }
    } while(1);
}

 int loop(int n){
    int i, j;
        
    for(i = 0; i < n; i++){
        
        
        for(j = 0; j < (n - i); j++){
            printf("   ");
        }
        
        for(j = 0; j <= i; j++){
            printf(" * ");
        }
        
        printf("\n");
    }
    
    return 0;
}

/**
 * Written by Jovan De Guia
 * Github Username: jxmked
 * */
